.github/workflows/python-app.yml

```diff
@@ -15,8 +15,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        django_version: [ '3.2' , '4.0', '4.1', '4.2']
-        python_version: [ '3.7', '3.8', '3.9' , '3.10', '3.11' ]
+        django_version:
+        - '3.2'
+        - '4.0'
+        - '4.1'
+        - '4.2'
+        - '5.0'
+        python_version:
+        - '3.7'
+        - '3.8'
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        - '3.12'
```

Comment:
splitting one per line will reduce the diff noise when adding/removing versions in the future